src/openapi.yaml

```diff
@@ -1738,11 +1738,10 @@ paths:
       parameters:
       - name: informatieobject
         in: query
-        description: URL naar de/het gerelateerde informatieobject
+        description: ''
```

Comment:
Dit is jammer. Komt door VNG API common change van declared filters neem ik aan.

Reply:
Denk dat het door de refactor komt met versionering en dat daar een help-text verdwenen is. Komt niet door deze changeset, maar de vorige waar een `generate-schema` vergeten is.

Reply:
ok, komt wel met de doorloop dan.